build.gradle

```diff
@@ -99,9 +100,9 @@ dependencies {
     testImplementation 'org.awaitility:awaitility'
 
     // mongo
-    testImplementation 'org.testcontainers:testcontainers:1.16.2'
-    testImplementation 'org.testcontainers:junit-jupiter:1.16.2'
-    testImplementation 'org.testcontainers:mongodb:1.16.2'
+    testImplementation "org.testcontainers:testcontainers:$testContainersVersion"
```

Comment:
This explicit dependency is removable, it comes through `org.testcontainers:junit-jupiter`

Reply:
Good catch! Please see https://github.com/iExecBlockchainComputing/iexec-core/pull/613/commits/dc6e604df3d38de61b594e03aaf1b5c6de28058a.